package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server ./lib/index.js"
+      "start": "webpack --watch ./lib/index.js ./js/bundle.js"
```

Comment:
Adjusted this script to run with the "normal" webpack command with the `--watch` flag. This means that it will updated every time you save a file that is inside of the folder you have it pointed to.